### PR TITLE
ENG-15466: Integration Tests for OCR Api using Sandbox for Api flow

### PR DESCRIPTION
**Pull request by satish-eligible**
|JIRA    | [ENG-15466](https://eligible.atlassian.net/browse/ENG-15466)
|:-------|:-----------------------------------------------------------

### Description

Ocr class added to Eligible module for integration tests.

### Checklists

**Development and testing:**
- [x] The code changed/moved as a part of this PR has been covered with tests
- [x] All tests related to the changed code pass in development
- [x] Pull request is ready for review

### Related Pull Requests

|project      | PR
|-------------|-----------------------
| gds_integration_tests | https://github.eligibleapi.com/EligibleAPI/gds_integration_tests/pull/75

### CODE REVIEW

File: lib/eligible/ocr.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module Eligible
+  class Ocr < APIResource
+    def self.setup_file(params)
```
Comment (ankit819) [inline lib/eligible/ocr.rb]: This can be made a private function.

File: lib/eligible/ocr.rb [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+      file = Util.value(params, :file)
+      params[:file] = File.new(file, 'rb') if file.is_a?(String)
```
Comment (ankit819) [inline lib/eligible/ocr.rb]: Does this work with Tempfile as well?

Comment (satish-eligible) [reply lib/eligible/ocr.rb]: nope, checked it, we are trying to read a existing file path, and Tempfile creation alters the name/path

File: lib/eligible/ocr.rb (hunk 1 above)
Comment (satish-eligible) [reply lib/eligible/ocr.rb]: done

**Comment by ankit819** [comment]
lgtm

**Pull request merged**